src/theme/mod.rs

```diff
@@ -126,4 +126,38 @@ mod tests {
         assert_eq!(got, should_be);
     }
 
+    #[test]
+    fn theme_dir_overrides_defaults() {
+        // Get all the non-Rust files in the theme directory
+        let special_files = PathBuf::from(env!("CARGO_MANIFEST_DIR"))
+            .join("src/theme")
+            .read_dir()
+            .unwrap()
+            .filter_map(|f| f.ok())
+            .map(|f| f.path())
+            .filter(|p| p.is_file() && !p.ends_with(".rs"));
+
+        let temp = TempDir::new("mdbook").unwrap();
+
+        // "touch" all of the special files so we have empty copies
+        for special_file in special_files {
+            let filename = temp.path().join(special_file.file_name().unwrap());
+            let _ = File::create(&filename);
+        }
+
+        let got = Theme::new(temp.path());
+
+        // if you can make this less repetitive, please do!
+        assert!(got.index.is_empty());
```

Comment:
This might get hard to keep in sync with the actual `Theme` struct contents.
We might make an additional empty constructor for Theme or just construct empty instance. It is minimally less repetitive (subjectively) and forces us to keep the test in sync.

how about?
```rust
        let empty = Theme {
            index: vec![],
            css: vec![],
            favicon: vec![],
            js: vec![],
            highlight_css: vec![],
            tomorrow_night_css: vec![],
            ayu_highlight_css: vec![],
            highlight_js: vec![],
            clipboard_js: vec![],
            store_js: vec![],
            jquery: vec![],
        };

        let got = Theme::new(temp.path());

        assert_eq!(got, empty);
```